src/roles/fracker.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+const MetaRole = require('roles_meta')
+
+class Fracker extends MetaRole {
+  getBuild (room, options) {
+    this.setBuildDefaults(room, options)
+    // 10 per harvest * 0.2 harvests per tick = 2 mineral per tick
```

Comment:
The extractor cooldown works out to harvesting every 6 ticks, make sure your math works out with this.